### DISCUSSION TITLE
How to turn off "always run as administrator "?

### DISCUSSION THREAD

**Discussion by Ran-oops** [PowerShell/PowerShell#17581] [Q&A]
Right now, My PowerShell always `run as administrator`. Is there a way to turn off?

Here are some trial:
- Windows Terminal default config: `run as admin` is **disabled**
- Windows Terminal PowerShell config: `run as admin` is **disabled**
- Powershell>Properties>Advanced: `run as admin` is **disabled**

After checking the above, the powershell still works as `admin`. Could someone give me some help? Thanks a lot.

**Comment by kasini3000** [comment]
C:\windows\System32\UserAccountControlSettings.exe and enable uac

if setting tool bad，try：
sfc /scannow
DISM.exe /Online /Cleanup-image /Scanhealth
DISM.exe /Online /Cleanup-image /Restorehealth

check your environment variable setting too

**Reply by Ran-oops** [reply]
Emmm, after running the above three commands,
- Powershell runs as non-admin
- However, the PC is broken(So many Programs throw this error: `Windows Cannot Access the Specified Device, Path or File`)

**So, I don't suggest you run the above commands.**
